### PR TITLE
Update README and clean up files.

### PR DESCRIPTION
**Pull request by hblumberg**
This diff:
1. Updates the README overview to include React and Next.
2. Cleans up some files:
a. Moves `geocode/geoLookup.js` to `utils/geo-lookup.js`.
b. Moves `pages/dev.js` to `pages/dev/index.js`
c. Removes the unused `misc/theme-flow.png`
3. Adds the folder structure section to the README to help new folks understand our two co-existing frontend frameworks.

### Test Plan:
Smoke test to confirm the website works as usual with no error logs.

Updated README:
Screen Shot 2021-02-15 at 1 23 25 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,4 +18,33 @@ open http://127.0.0.1:3002/
...
+├── postman
+├── routes
+├── static
+│   ├── bootstrap
+│   └── jquery
+├── styles
+├── utils
+├── views
+└── app.js
+```
```
Comment (hblumberg) [inline README.md]: I could remove this section and just keep the pieces below?

Comment (johndeyrup) [reply README.md]: yeah not sure how much value it adds; and we would need to keep it updated

**Comment by johndeyrup** [comment]
lgtm

**Review by johndeyrup — APPROVED** [review]

**Review by mzagaja — APPROVED** [review]
I couldn't test the geolocation locally due to https restrictions, but everything else looks good. Ready to merge!

**Pull request merged**